gfc/financial_companion/tests/views/test_all_groups_view.py

```diff
@@ -47,7 +47,7 @@ def test_post_when_partial_group_name_is_applied(self):
         self.assertEqual(len(messages_list), 0)
         self.assertContains(response, "Saving Club")
         self.assertContains(response, "Save with us")
-        self.assertContains(response, "SavingsRUs")
+        self.assertContains(response, "SavingsRUs".title())
```

Comment:
What happens if the user wants to have the name of their group displayed as they wrote it? @Splagon

Reply:
I can now foresee the results of my actions and have taken action to remove capitalisation from the individual category/account/transaction/group views